sympy/series/formal.py

```diff
@@ -810,6 +811,13 @@ def _compute_fps(f, x, x0, dir, hyper, order, rational, full):
             return ak, xk, ind
         return None
 
+    syms = list(f.free_symbols.difference({x}))
+    symb = S.One
+    if len(syms) == S.One:
+        (t1, t2) = expand(f).as_independent(syms[0])
```

Comment:
Is it possible to directly write, `(f, symb) = expand(f).as_independent(syms[0])` if `t1` and `t2` are not required anywhere in this file?